### ISSUE TITLE
Dectalk crashes after speech.

### ISSUE THREAD

**Issue by AdamT567** [Frodeluren/DECtalk-Twitch-TTS-bot#10]
After speech is redeemed through channel points the dectalk console force closes itself. Have tried reinstalling everything to try and fix any hiccups but nothing has worked. Unsure why it is doing it now when it was fine before. Also is there a log file I am overlooking, or does one not exist?

**Comment by frodekp** [comment]
There is currently not any log file. 

How do you go about when running it? 
The terminal shouldn't close on crash if you open the terminal first before running the python script. 
If you run the script by double click it, it will close on crash. 
Can you try opening the terminal first, then go to the project path and then use `python dectalktwitchtts.py` and see if you can get error message? 

I tried running it with only CHANNEL_POINT_REDEMPTION_ALLOWED enabled but didn't crash when a valid message got trough. So I'm not sure what the error could be.

**Comment by AdamT567** [comment]
So I followed your instructions and got an `Input: Segmentation fault`. I'll attach a screenshot of my terminal to this comment as well. 

![image](https://github.com/frodekp/DECtalk-Twitch-TTS-bot/assets/86427384/8e4bbfbc-4560-4d03-88ef-f856f60cdb4a)

**Comment by SupaPhly** [comment]
I'm having the same problem, as soon as it speaks the first message, the script stops

**Comment by frodekp** [comment]
I cant seem to reproduce this error. What python version are you using? `python --version`

**Comment by SupaPhly** [comment]
> I cant seem to reproduce this error. What python version are you using? `python --version`

Python 3.12.3

should I be using another version?

**Comment by frodekp** [comment]
I just tested it with Python 3.12.3 and got the same error as you did. Earlier I used Python 3.11.3 when it worked for me.

**Comment by AdamT567** [comment]
> I just tested it with Python 3.12.3 and got the same error as you did. Earlier I used Python 3.11.3 when it worked for me.

yeah I'm on 3.12.2 which is what it installed just a few months ago. I had moved to a new, fresh install of windows on a second computer and ran the install script provided.
